web/src/components/Layout.tsx

```diff
@@ -117,24 +117,25 @@ type PropsType = {
  * If a footer is supplied and there's not enough content (in header and children) to fill the viewbox, the footer will
  * always stick to the bottom of the viewbox.
  */
-const Layout = ({
-  asideStickyTop = 0,
-  footer,
-  header,
-  toolbar,
-  modal,
-  children,
-  fullWidth = false
-}: PropsType): JSX.Element => {
+const Layout = ({ footer, header, toolbar, modal, children, fullWidth = false }: PropsType): JSX.Element => {
   const modalVisible = !!modal
+  const { width } = useWindowDimensions()
+  const [languageSelectorHeight, setLanguageSelectorHeight] = useState<number>(0)
+
+  useEffect(() => {
+    const panelHeight = document.getElementById('languageSelector')?.clientHeight
+    if (panelHeight) {
+      setLanguageSelectorHeight(panelHeight)
+    }
```

Comment:
Shouldn't this rather be:
```suggestion
    setLanguageSelectorHeight(panelHeight ?? 0)
```